src/ui/templates/insurance/application-submitted.njk

```diff
@@ -51,7 +51,9 @@
         <span data-cy="help-us-improve-if-you-like">{{ CONTENT_STRINGS.HELP_US_IMPROVE.IF_YOU_LIKE }}</span>
 
         <span data-cy="help-us-improve-feedback-intro">{{ CONTENT_STRINGS.HELP_US_IMPROVE.FEEDBACK.INTRO }}</span>
-        <a class="govuk-link govuk-link--no-visited-state" href="{{ CONTENT_STRINGS.HELP_US_IMPROVE.FEEDBACK.LINK.HREF }}" data-cy="help-us-improve-feedback-link">{{ CONTENT_STRINGS.HELP_US_IMPROVE.FEEDBACK.LINK.TEXT }}</a>
+        <span data-cy="help-us-improve-feedback-text">
+          <a class="govuk-link govuk-link--no-visited-state" href="{{ CONTENT_STRINGS.HELP_US_IMPROVE.FEEDBACK.LINK.HREF }}" data-cy="help-us-improve-feedback-link">{{ CONTENT_STRINGS.HELP_US_IMPROVE.FEEDBACK.LINK.TEXT }}</a>.
```

Comment:
Did consider this, I think it's better not to. Reason being because this generic "give feedback" text can be used in other places that may or may not want a full stop.

We actually have 2x instances now of different "give feedback" text which might be aligned with or without full stops.